examples/DataApi-Voting-poll/scripts/send_data_transaction_create_poll.js

```diff
@@ -0,0 +1,101 @@
+const { dag4 } = require( '@stardust-collective/dag4' );
```

Comment:
Can we put all of the sending functions into a single file? There's a lot of repeated code otherwise.

Reply:
Done